src/hb.hh

```diff
@@ -499,14 +499,12 @@ static_assert ((sizeof (hb_var_int_t) == 4), "");
 #define HB_VAR_ARRAY 1
 #endif
 
-static inline double
+static inline float
 _hb_roundf (float x)
 {
-  return x >= 0 ? floor ((double) x + .5) : ceil ((double) x - .5);
+  return ceilf (x - .5f);
```

Comment:
This rounds halfway values down, not up. I think it should be `floorf (x + .5f)`.

Reply:
Thanks. The tests pass now.